trazip/settings.py

```diff
@@ -34,6 +34,51 @@
 
 # Application definition
 
+#log configure
+LOGGING = {
```

Comment:
I already define LOGGING in this file.
Please move the config about how to output the log in file to the existing LOGGING.